libs/bits/bit_array.go

```diff
@@ -255,6 +256,13 @@ func (bA *BitArray) IsFull() bool {
 	return (lastElem+1)&((uint64(1)<<uint(lastElemBits))-1) == 0
 }
 
+func (bA *BitArray) getTrueIndicesBufferLocked() []int {
```

Comment:
can we add correctness tests for this and/or assert correct values for the output of this method?

also I think "locked" is unclear given that there is no lock (I think adding "Unsafe" to these method names is maybe better?)